backend/src/main/resources/application.yaml

```
@@ -19,6 +19,8 @@ spring:
   jackson:
     serialization:
       FAIL_ON_EMPTY_BEANS: false
+  session:
+    store-type: "jdbc"
```

No need for the quotes here.
```suggestion
    store-type: jdbc
```